probatus/feature_elimination/feature_elimination.py

```diff
@@ -13,7 +13,7 @@
     preprocess_labels,
     shap_calc,
```

Comment:
Could you add a single simple unit test for regression and one for multi class?
Would be good to make a very simple rows 4 4 columns dataset and write a simple test, where you can by hand ensure that the correct feature is picked by a simple model.

Reply:
added, sorry for the delay